nix/pkgs/php81/php.ini

```diff
@@ -0,0 +1,29 @@
+; Note: This file is automatically appended to the php.ini.
+; If you update, be sure to restart any nix shells or processes.
+
+date.timezone = America/Los_Angeles
+; date.timezone = Europe/London
+
+max_execution_time = -1
+memory_limit = 256M
+
+xdebug.mode=debug
+xdebug.start_with_request=yes
+xdebug.remote_handler=dbgp
+xdebug.client_host=localhost
+xdebug.client_port=9000
+xdebug.log_level=0
+xdebug.discover_client_host=1
+xdebug.mode=develop
```

Comment:
minor note - should either combine with line 10 (`xdebug.mode=debug,develop`) or remove line 10. If this is mostly for unit tests they'll run faster without "debug" and I don't think it's needed for those runs(?)